src/spyglass/spikesorting/v1/sorting.py

```diff
@@ -242,18 +243,40 @@ def make(self, key: dict):
             # Specify tempdir (expected by some sorters like mountainsort4)
             sorter_temp_dir = tempfile.TemporaryDirectory(dir=temp_dir)
             sorter_params["tempdir"] = sorter_temp_dir.name
+            os.chmod(temp_dir_path, 0o777)
```

Comment:
We should probably poll the lab / resolve our discussion in https://github.com/LorenFrankLab/spyglass/issues/592?

Reply:
If we can set up the unix groups so that `774` works, I'd prefer that

Reply:
@donghoon-shin can you change this per suggestion from @CBroz1 ?

Reply:
Done!